### PR TITLE
[Nokia][Nokia-IXR7250E-36x400G] Updated BCM config file to disable the cmic dma abort during cold boot

### PR DESCRIPTION
**Pull request by saksarav-nokia**
Signed-off-by: Sakthivadivu Saravanaraj <sakthivadivu.saravanaraj@nokia.com>

#### Why I did it
syncd was crashing after autorestart oc test suite with segmentation fault in BCM SDK when it aborts the cmic dma.

#### How I did it
 Since the cmic dma abort is happening during cold boot and told by broadcom, it is safe to disable the cmic dma abort for cold boot with soc property, added the soc flag in .bcm config file
#### How to verify it
Verified autorestart oc test suite passes with the fix.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

File: device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x400G/0/jr2cp-nokia-18x400g-config.bcm [hunk 1]
```diff
@@ -2055,3 +2055,4 @@ modreg IPS_FORCE_LOCAL_OR_FABRIC FORCE_FABRIC=1
 
 xflow_macsec_secure_chan_to_num_secure_assoc_encrypt=2
 xflow_macsec_secure_chan_to_num_secure_assoc_decrypt=4
+cmic_dma_abort_in_cold_boot=0
```
Comment (judyjoseph) [inline device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x400G/0/jr2cp-nokia-18x400g-config.bcm:2058]: Can you add this parameter to src/sonic-device-data/tests/permitted_list

Comment (saksarav-nokia) [reply device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x400G/0/jr2cp-nokia-18x400g-config.bcm:2058]: Judy, I have added to  src/sonic-device-data/tests/permitted_list. Please check

**Comment by saksarav-nokia** [comment]
Hi Judy, I updated BCM config with correct header settings for recyle port used for Everflow to make the everflow work. Please review it and merge the PR

**Comment by judyjoseph** [comment]
@saksarav-nokia could you resolve the conflicts

**Comment by saksarav-nokia** [comment]
@judyjoseph , the conflicts are resolved now

**Review by judyjoseph — APPROVED** [review]

**Comment by judyjoseph** [comment]
@saksarav-nokia could you raise a separate PR for 202205 as there is conflict

**Comment by judyjoseph** [comment]
@saksarav-nokia could you raise a separate PR for 202205

**Pull request merged**